common/cuda_hip/preconditioner/batch_jacobi.hpp.inc

```diff
@@ -77,7 +112,9 @@ public:
                  j < mat.row_ptrs[irow + 1]; j += warp_size) {
                 const int found = (mat.col_idxs[j] == irow);
                 if (found) {
-                    work_[irow] = one<ValueType>() / mat.values[j];
+                    work_[irow] = (mat.values[j] != zero<ValueType>())
```

Comment:
I think it is mainly a verification to make sure we don't get any inf's. Do you think there is any specific advantage of your approach ? I doubt we will have entries that are < numeric_limits and not zero because we kind of sanitize in read and so on ?

Reply:
Basically I thought this was related to out recent discussion about numerical issues with Jacobi.